mayan/apps/appearance/templates/appearance/generic_list_items_dropdown_subtemplate.html

```diff
@@ -0,0 +1,95 @@
+{% load i18n %}
+{% load static %}
+
+{% load common_tags %}
+{% load navigation_tags %}
+
+<div class="row">
+    <div class="col-xs-12">
+        {% if not object_list %}
+            <div class="well">
+                {% include 'appearance/no_results.html' %}
+            </div>
+        {% else %}
+            {% include 'appearance/list_header.html' %}
+            {% navigation_resolve_menu name='multi item' sort_results=True source=object_list.0 as links_multi_menus_results %}
+            <div class="well center-block">
+                <div class="row row-items">
+                    {% for object in object_list %}
+                        <div class="{{ column_class|default:'col-xs-12 col-sm-4 col-md-3 col-lg-2' }}">
+                            <div class="panel panel-primary panel-item">
+                                <div class="panel-heading">
+                                    <div class="form-group">
+                                        <div class="checkbox">
+                                            <label for="id_indexes_0" style="cursor: auto;">
+                                                {% if links_multi_menus_results and not hide_multi_item_actions %}
+                                                    <input class="form-multi-object-action-checkbox check-all-slave checkbox" name="pk_{{ object.pk }}" style="cursor: pointer;" type="checkbox" />
+                                                {% endif %}
+
+                                                <span style="color: white; word-break: break-all; overflow-wrap: break-word;">
+                                                    {% if not hide_object %}
+                                                        {% if not hide_link %}
+                                                            <a href="{{ object.get_absolute_url }}">{{ object }}</a>
+                                                        {% else %}
+                                                            {{ object }}
+                                                        {% endif %}
+                                                    {% else %}
+                                                        {% navigation_get_source_columns source=object only_identifier=True as source_column %}
+                                                        {% navigation_source_column_resolve column=source_column as column_value %}
+                                                        {{ column_value }}
+                                                    {% endif %}
+                                                    <select>

```

Comment:
Lines 41-46 are where the dropdown lives. @Yu-Zhou-Wang the value attribute is how you will find out which reviewer was selected. You will likely also need to extract the document file name so you can modify it's ACL with reference to this user.